certs/service.go

```diff
@@ -34,18 +26,13 @@ var (
 	// when accessing a protected resource.
 	ErrUnauthorizedAccess = errors.New("missing or invalid credentials provided")
 
-	errFailedKeyCreation         = errors.New("failed to create client private key")
-	errFailedDateSetting         = errors.New("failed to set date for certificate")
-	errKeyBitsValueWrong         = errors.New("missing RSA bits for certificate creation")
-	errMissingCACertificate      = errors.New("missing CA certificate for certificate signing")
-	errFailedSerialGeneration    = errors.New("failed to generate certificate serial")
-	errFailedPemKeyWrite         = errors.New("failed to write PEM key")
-	errFailedPemDataWrite        = errors.New("failed to write pem data for certificate")
-	errPrivateKeyUnsupportedType = errors.New("private key type is unsupported")
-	errPrivateKeyEmpty           = errors.New("private key is empty")
-	errFailedToRemoveCertFromDB  = errors.New("failed to remove cert serial from db")
-	errFailedCertCreation        = errors.New("failed to create client certificate")
-	errFailedCertRevocation      = errors.New("failed to revoke certificate")
+	// ErrFailedCertCreation failed to create certificate.
```

Comment:
You can put a single comment explanation above this `var` block to avoid writing unnecessary comments just to satisfy linter.